### PR TITLE
[11.0] [mig] purchase triple discount

### PR DESCRIPTION
**Pull request by sferket**

### CODE REVIEW

File: purchase_triple_discount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Purchase Triple Discount',
+    'version': '11.0.1.1.0',
+    'category': 'Purchase Management',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://tecnativa.com',
```
Comment (nikul-serpentcs) [inline purchase_triple_discount/__manifest__.py:9]: @sferket Replace website URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: purchase_triple_discount/tests/test_purchase_discount.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+class TestPurchaseOrder(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestPurchaseOrder, cls).setUpClass()
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Mr. Odoo',
+        })
+        cls.product1 = cls.env['product.product'].create({
```
Comment (nikul-serpentcs) [inline purchase_triple_discount/tests/test_purchase_discount.py:15]: here `cls.env['product.product']` use multiple time, so you should use as a global variable.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: purchase_triple_discount/models/purchase_order.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    discount2 = fields.Float(
+        'Disc. 2 (%)',
+        digits=dp.get_precision('Discount'),
+        default=0.0,
+    )
+
+    discount3 = fields.Float(
+        'Disc. 3 (%)',
+        digits=dp.get_precision('Discount'),
+        default=0.0,
```
Comment (anandkansagra) [inline purchase_triple_discount/models/purchase_order.py:24]: I think we can remove `default=0.0`

File: purchase_triple_discount/models/purchase_order.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    _inherit = "purchase.order.line"
+
+    @api.depends('discount2', 'discount3')
+    def _compute_amount(self):
+        super(PurchaseOrderLine, self)._compute_amount()
+
+    discount2 = fields.Float(
+        'Disc. 2 (%)',
+        digits=dp.get_precision('Discount'),
+        default=0.0,
```
Comment (anandkansagra) [inline purchase_triple_discount/models/purchase_order.py:18]: I think we can remove `default=0.0`

File: purchase_triple_discount/views/purchase_view.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (anandkansagra) [inline purchase_triple_discount/views/purchase_view.xml]: No need of `encoding="utf-8"`

**Review by anandkansagra — CHANGES_REQUESTED** [review]

File: purchase_triple_discount/views/purchase_view.xml (hunk 5 above)
Comment (pedrobaeza) [inline purchase_triple_discount/views/purchase_view.xml:1]: Yes, it's needed in XML files.

Comment (anandkansagra) [reply purchase_triple_discount/views/purchase_view.xml:1]: Thanks @pedrobaeza ,
LGTM

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by pedrobaeza** [comment]
And check CI status

**Review by anandkansagra — APPROVED** [review]

**Comment by liebana** [comment]
Which one is the correct one?

https://github.com/OCA/purchase-workflow/pull/611

**Comment by pedrobaeza** [comment]
Both PRs (#595 and #611) are in red state, so no good choice for now.

**Comment by rafaelbn** [comment]
Hello @sferket , thanks for this contribution.

It's 7 month since this PR without answering, are you going to continue or if not can we close it?

Thanks in advance!

**Comment by pedrobaeza** [comment]
Superseeded by #732

**Pull request closed without merging**